recognition/scoring.py

```diff
@@ -202,12 +202,21 @@ def calc_wer(self):
 
 
 class Hub5ScoreJob(Job):
-    def __init__(self, ref, glm, hyp):
+    __sis_hash_exclude__ = {"sctk_binary_path": None}
+
+    def __init__(
+        self,
+        ref: Union[tk.Path, str],
+        glm: Union[tk.Path, str],
+        hyp: Union[tk.Path, str],
+        sctk_binary_path: Optional[tk.Path] = None,
+    ):
```

Comment:
```suggestion
    ):
        """
        :param ref: reference stm text file
        :param glm: text file containing mapping rules for scoring
        :param hyp: hypothesis ctm text file
        :param sctk_binary_path: set an explicit binary path.
        """
```
If we are at it, let me add a doc string.